app/lib/actions/katello/organization/manifest_import.rb

```diff
@@ -0,0 +1,40 @@
+module Actions
+  module Katello
+    module Organization
+      class ManifestImport < Actions::AbstractAsyncTask
+        middleware.use Actions::Middleware::PropagateCandlepinErrors
+
+        def plan(organization, path, force)
+          action_subject organization
+          manifest_update = organization.products.redhat.any?
+
+          sequence do
+            plan_action(Candlepin::Owner::Import,
+                        :label => organization.label,
+                        :path => path,
+                        :force => force)
+            plan_action(Candlepin::Owner::ImportProducts, :organization_id => organization.id)
+
+            if manifest_update && SETTINGS[:katello][:use_pulp]
+              organization.products.redhat.flat_map(&:repositories).each do |repo|
+                plan_action(Katello::Repository::RefreshRepository, repo)
+              end
+            end
+          end
+        end
+
+        input_format do
+          param :organization, Hash do
+            param :id
+          end
+          param :path
+          param :force
+        end
```

Comment:
What's the reasoning behind putting input_format in the middle here? rather than at the top?

This import format isn't actually correct (and really isn't needed if there is a plan method